src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/Server/SqlNorm.cs

```diff
@@ -16,32 +16,34 @@ namespace Microsoft.Data.SqlClient.Server
     // a particular field.
     internal sealed class FieldInfoEx : IComparable
     {
-        internal readonly int Offset;
-        internal readonly FieldInfo FieldInfo;
-        internal readonly Normalizer Normalizer;
+        private readonly int _offset;
 
         internal FieldInfoEx(FieldInfo fi, int offset, Normalizer normalizer)
         {
-            FieldInfo = fi;
-            Offset = offset;
             Debug.Assert(normalizer != null, "normalizer argument should not be null!");
             Normalizer = normalizer;
+            FieldInfo = fi;
+            _offset = offset;
         }
+        public FieldInfo FieldInfo { get; private set; }
+        public Normalizer Normalizer { get; private set; }
```

Comment:
Any special reason to make these props public?
They were internal in both projects.

Reply:
Personal preference. I've changed it. 

Those members are accessed by users of the class and in those cases I prefer to use public indicating that they're part of the public contract for the type. In general my preference is to mark types as internal and members of those types as public private protected as you would a publicly visible class. I'll follow the convention for the project as always though.